src/pages/LearnPage.tsx

```diff
@@ -47,16 +57,96 @@ const NextButton = styled(IconButton)({
   },
 })
 
+type scrollVisibilityApiType = React.ContextType<typeof VisibilityContext>
+
+function onWheel(apiObj: scrollVisibilityApiType, ev: React.WheelEvent): void {
+  const isThouchpad = Math.abs(ev.deltaX) !== 0 || Math.abs(ev.deltaY) < 15
+
+  if (isThouchpad) {
+    ev.stopPropagation()
+    return
+  }
+
+  if (ev.deltaY < 0) {
+    apiObj.scrollNext()
+  } else if (ev.deltaY > 0) {
+    apiObj.scrollPrev()
+  }
+}
+
 interface LessonGroupProps {
   title: string
-  children: React.ReactNode
+  lessons: LessonData[]
 }
 
-function LessonGroup({ title, children }: LessonGroupProps) {
+// itemId is needed for tracking items in the horizontal scroll list
+// eslint-disable-next-line
+function Lesson({ lesson, itemId }: { lesson: LessonData; itemId: string }) {
+  return (
+    <>
+      <LessonButton sx={{ mx: '10px', whiteSpace: 'nowrap' }}>
+        <Link
+          style={{ color: 'inherit', textDecoration: 'none' }}
+          to={'/learn/' + lesson.urlParam}
+        >
+          {lesson.name}
+        </Link>
+      </LessonButton>
+    </>
+  )
+}
+
+function LeftArrow() {
+  const visibility = React.useContext<publicApiType>(VisibilityContext)
```

Comment:
I don't understand what 'publicApiType' does, but I trust you. Seems like a placeholder.

Reply:
Copied code. But it's the type of the visibility context, which tells us if the button should be disabled.